### ISSUE TITLE
Error on hide/show pane with v-if

### ISSUE THREAD

**Issue by calebeaires** [antoniandre/splitpanes#73]
Version: 2.1.2

The error
**Error in callback for watcher "panes": "TypeError: Cannot read property 'update' of undefined"**

Code position: 

```javascript
 updatePanesStyle: function updatePanesStyle() {
      var _this = this;

      // Using `this.$children` here rather than `this.$slots.default` because the latter is sometimes not initialized yet (eg. when this method is called
      // whereas the component is not mounted yet).
      var children = this.$children;
      this.panes.forEach(function (pane) {
        children[pane.index].update(_defineProperty({}, _this.horizontal ? 'height' : 'width', "".concat(pane.size, "%")));
      });
    }
```
More info:
Using nested panes

**Comment by antoniandre** [comment]
Hi @calebeaires,
Thanks for reporting, this is actually the same bug as #67 (related to v-if), I am currently working on a fix for that and it will fix both issues.
Please continue the discussion in #67 if you need to,
I will update in there when the fix is ready.

**Comment by antoniandre** [comment]
Should now be fixed (same as #67).
